client/landing/stepper/declarative-flow/tailored-ecommerce-flow.ts

```diff
@@ -1,42 +1,68 @@
+import { useLocale } from '@automattic/i18n-utils';
+import { useFlowProgress, ECOMMERCE_FLOW } from '@automattic/onboarding';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { useEffect } from 'react';
+import { recordFullStoryEvent } from 'calypso/lib/analytics/fullstory';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { USER_STORE, ONBOARD_STORE } from '../stores';
 import { recordSubmitStep } from './internals/analytics/record-submit-step';
 import type { StepPath } from './internals/steps-repository';
 import type { Flow, ProvidedDependencies } from './internals/types';
 
-export const tailoredEcommerceFlow: Flow = {
-	name: 'tailored-ecommerce',
-
+export const ecommerceFlow: Flow = {
+	name: ECOMMERCE_FLOW,
 	useSteps() {
-		return [ 'storeProfiler' ] as StepPath[];
+		useEffect( () => {
+			recordTracksEvent( 'calypso_signup_start', { flow: this.name } );
+			recordFullStoryEvent( 'calypso_signup_start_ecommerce', { flow: this.name } );
+		}, [] );
+
+		return [ 'intro', 'storeProfiler' ] as StepPath[];
 	},
 
-	useStepNavigation( currentStep, navigate ) {
+	useStepNavigation( _currentStep, navigate ) {
```

Comment:
Nit: I am not sure if we're doing this in other code, but I'd prefer this to be `_currentStepName` or `currentStepName`, as just `_currentStep` is not obviously a string vs a reference to the step object. This might also be helped if the arguments were typed.

Reply:
Some of this is based on existing patterns established elsewhere in stepper but I will do some tweaking on this.